backend/src/db/controllers/users.js

```diff
@@ -12,4 +12,21 @@ export async function getUserByEmail(email) {
     return await User.findOne({ "email": email})
 }
 
+export async function getRoadTripsAttending(userId) {
+    const dbUser = await User.findById(userId);
+    await User.populate(dbUser, 'roadTripsAttending');
+    return dbUser.roadTripsAttending;
+}
+
+export async function getRoadTripsOrganising(userId) {
+    const dbUser = await User.findById(userId);
+    await User.populate(dbUser, 'roadTripsOrganising');
+    return dbUser.roadTripsOrganising;
+}
+
+export async function addRoadTripsOrganising(roadTripId, userId) {
```

Comment:
completed and utter nit: can we change the method signature to `export async function addRoadTripOrganising(userId, roadTripId) {` just for clarity and consistency